packages/integrations/stores/ini-file/package.json

```
@@ -0,0 +1,35 @@
+{
+    "name": "@configu-integrations/ini-file",
+    "version": "0.0.0",
+    "license": "Apache-2.0",
+    "author": "@configu/dev",
+    "files": [],
```

Just did some testing, you'll need to change this to:
```suggestion
    "main": "src/ini-file.ts",
```
in order for this to successfully register as an integration

---

Actually most of the other integrations have `"files": [],`, so maybe this isn't actually needed. Not sure why it was failing for me during testing but I'd say feel free to disregard this suggestion for the time being.

If it fails to register we can take a look as part of a separate issue, but for the time being I'd say consistency is better.